packages/notification-center/src/components/notification-center/components/AppContent.tsx

```diff
@@ -22,22 +20,21 @@ export function AppContent() {
     }
   );
 
-  const theme = {
+  const themeConfig = {
     colors: {
-      main: organization?.branding?.color || colors.vertical,
-      fontColor: colorScheme === 'light' ? colors.B40 : colors.white,
-      secondaryFontColor: colorScheme === 'light' ? colors.B80 : colors.B40,
+      main: theme.loaderColor || organization?.branding?.color,
```

Comment:
@scopsy Is this what you meant?

In addition at the current state, I am not sure there is a purpose of the organization configuration because we use here the theme which is or defined by the user via the code or the defaults that there are so there won't be any fallback to UI config, what do you think?

Reply:
We might leave this out then on the UI part later. Let's leave it like this for now. It's great.